Gemfile

```diff
@@ -4,9 +4,10 @@ source 'https://rubygems.org'
 ruby '2.5.0'
 
 gem 'donation_system', git: 'https://github.com/survival/donation-system',
-                       tag: 'v1.0.0'
+                       branch: 'paypal-one-off-salesforce'
```

Comment:
I presume this will be changed back before merging?

Reply:
yes